### ISSUE TITLE
Implement custom font color into markup

### ISSUE THREAD

**Issue by tishka25** [johnoneil/LEDSign#2]
Hello,

Your project has helped me allot, so I want to thank you for that!

Also I have made some changes to the **JetFileIIProtocol.py** file. It supports now custom RGB or BGR font color.

### Usage example
```
msg = TextFile('{top}{center}{raindropsin}{font_color_rgb=30,167,124}I am in custom font color', "AB.nmg", drive='E')
serial.write(msg)
```

I would be very happy if you accept this contribution!

**Comment by johnoneil** [comment]
Yeah sure @tishka25 . Appreciate it. Just send the change as a Pull Request and I'll merge it.
